src/components/snapshot/overview/SnapshotOverviewPanel.tsx

```diff
@@ -154,6 +159,11 @@ function SnapshotOverviewPanel(props: SnapshotOverviewPanelProps) {
             </Grid>
           )}
         </Grid>
+        {isSteward && (
+          <Grid item xs={12} className={classes.accordionWorkspaces}>
```

Comment:
Same shift of being clear on why this component will be rendered at the point of rendering it.

Reply:
Thanks, appreciate the extra clarity around your thought process.